generators/generator-base-private.js

```diff
@@ -865,8 +866,13 @@ module.exports = class extends Generator {
             this.warning(msg);
             return undefined;
         }
-        // eslint-disable-next-line global-require,import/no-dynamic-require
-        return require(path.join(blueprintPackagePath, 'package.json'));
+        try {
+            return JSON.parse(fs.readFileSync(path.join(blueprintPackagePath, 'package.json')));
+        } catch (_) {
+            const msg = `Could not retrieve package.json of blueprint '${blueprintPkgName}'`;
+            this.warning(msg);
+            return undefined;
+        }
```

Comment:
this too, tougher but still needed IMO

Reply:
This isn't supposed to happen, is just a safeguard.
Needs to mock a node package (blueprint) missing package.json.
Anyway returning undefined is correct.

Reply:
Not supposing to happen means it can happen right? ;)

Reply:
Happens in 2 cases:
* Bug in the blueprint.
* Bug in the environment.

Both cases doesn't need to be tested.

I think I will remove try/catch.
We really don't need this safe guard.